templates/frontend/objects/announcement_summary.tpl

```diff
@@ -23,9 +23,9 @@
 	<div class="announcement-summary-date">
 		{$announcement->getDatePosted()|date_format:$dateFormatLong}
 	</div>
-	<div class="announcement-summary-description">
+	<p class="announcement-summary-description">
 		{$announcement->getLocalizedDescriptionShort()|strip_unsafe_html}
-		<div class="announcement-summary-more">
+		<p class="announcement-summary-more">
```

Comment:
A `<p>` should not be nested inside of another `<p>`. I'm also not sure if it's valid to nest a `<div>` inside of a `<p>`. You probably want the outer `<p>` to be  a `<div>`. Or to do something like this:

```
<div>
  <p>...description...</p>
  <p><a>...</a></p>
</div>
```

Reply:
Oops! Thanks for catching that. I will fix it now.

Reply:
rebased + force-pushed!